src/Grpc.AspNetCore.Server/GrpcEndpointRouteBuilderExtensions.cs

```diff
@@ -90,20 +90,25 @@ public static class GrpcEndpointRouteBuilderExtensions
             return new CompositeEndpointConventionBuilder(serviceBinder.EndpointConventionBuilders);
         }
 
-        private static void ReflectionBind<TService>(ServiceBinderBase binder, TService service)
+        private static Type GetServiceBaseType(Type serviceImplementation)
```

Comment:
Was the point to share this logic between here and ReflectionGrpcServiceActivator.cs?

Reply:
Yeah